### PR TITLE
[add page] Triducer page

### PR DESCRIPTION
**Pull request by brunoolivieri**
This PR tries to fix issue #1673 by adding a brief of what I got from the forum: https://discuss.ardupilot.org/t/affordable-transducer-for-autonomous-boat/39484

@rmackay9 please, could you review that? Is it worth inserting something so brief?

I rendered all wiki in Vagrant I did not find link errors. I also did the same classification as the last Sonar inserted, which means, appears in the rover, copter, and plane wikis.

### CODE REVIEW

**Comment by rmackay9** [comment]
Thanks for this, I'm wondering how this fits in with this existing page: http://ardupilot.org/rover/docs/common-underwater-sonar-analog.html

File: common/source/docs/common-rangefinder-landingpage.rst [hunk 1]
```diff
@@ -67,6 +67,7 @@ based upon your set-up.
     TeraRanger One Rangefinder <common-teraranger-one-rangefinder>
     TerraRanger Tower (360 degree) <common-teraranger-tower-objectavoidance>
     Underwater Sonar (Analog) <common-underwater-sonar-analog>
+	Triducer <common-triducer>
```
Comment (khancyr) [inline common/source/docs/common-rangefinder-landingpage.rst]: tabs here

File: common/source/docs/common-triducer.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.. _common-triducer:
+
+[copywiki destination="copter,plane,rover"]
+
+=======================
+Triducer
+=======================
```
Comment (khancyr) [inline common/source/docs/common-triducer.rst]: too long ! it should match the text, same for the upper one's

File: common/source/docs/common-triducer.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Triducer combines Speed, temperature and Depth into one Transducer.
+
+Mounting the triducer
+----------------------
```
Comment (khancyr) [inline common/source/docs/common-triducer.rst]: should be check if the lenght it right, otherwise, it will trigger an annoying warning

**Comment by brunoolivieri** [comment]
@kd0aij  I believe that I fixed the warnings.

@rmackay9 I put down a level for underwater sonars which have the fish finder and the Triducer. Looks better?

File: common/source/docs/common-rangefinder-landingpage.rst (hunk 1 above)
Comment (brunoolivieri) [reply common/source/docs/common-rangefinder-landingpage.rst]: I erase such insertion.

File: common/source/docs/common-triducer.rst (hunk 2 above)
Comment (brunoolivieri) [reply common/source/docs/common-triducer.rst]: ok, thanks

File: common/source/docs/common-triducer.rst (hunk 3 above)
Comment (brunoolivieri) [reply common/source/docs/common-triducer.rst]: ok, I guess I correct that also.

**Comment by Hwurzburg** [comment]
Bruno, run spelling checker.....also, do you know how to squash the commits?

**Comment by brunoolivieri** [comment]
ok

done :-)

**Comment by brunoolivieri** [comment]
Thanks, Henry. But I already rendered/built the files before commit. 

Regarding the squashing and spell, I guess I did correctly this time. I think we can remove WIP tag and leave to be reviewed and possibly merged.

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**